mod_reforged/hooks/ai/tactical/behaviors/ai_attack_throw_net.nut

```diff
@@ -0,0 +1,10 @@
+::Reforged.HooksMod.hook("scripts/ai/tactical/behaviors/ai_attack_throw_net", function (q) {
+	q.onEvaluate = @(__original) function( _entity )
+	{
+		local ret = __original(_entity);
+		if (_entity.getSkills().hasSkill("perk.rf_kingfisher") && ::Math.rand(1, 100) < 75)
```

Comment:
the two conditions should be reversed for performance reasons

Reply:
Done